ddtrace/contrib/django/utils.py

```diff
@@ -85,9 +87,23 @@ def get_request_uri(request):
             log.debug("Failed to build Django request host", exc_info=True)
             host = "unknown"
 
+    # If request scheme is missing, possible in case where wsgi.url_scheme
+    # environ has not been set, then default to http
+    scheme = request.scheme or "http"
```

Comment:
Is it safe to assume http? I know in maybe like 99% of cases it's probably http, but if there's even a chance that it's not I'd rather we don't make the assumption.